packages/elements-core/src/components/Docs/HttpService/ServerInfo.tsx

```diff
@@ -27,24 +36,25 @@ export const ServerInfo: React.FC<ServerInfoProps> = ({ servers, mockUrl }) => {
       <Panel rounded isCollapsible={false} className="BaseURLContent" w="full">
         <Panel.Titlebar whitespace="nowrap">API Base URL</Panel.Titlebar>
         <Box overflowX="auto">
-          <Panel.Content w="max" className="sl-flex sl-flex-col">
-            {showProductionUrl && (
-              <Box whitespace="nowrap">
-                {showMocking && (
-                  <Text pr={2} fontWeight="bold">
-                    Production:
-                  </Text>
-                )}
-                <Text aria-label="production-server">{productionUrl}</Text>
+          <Panel.Content w="full" className="sl-flex sl-flex-col">
+            {serversToDisplay.map(({ name, url }) => (
+              <Box whitespace="nowrap" mb={1} key={url}>
```

Comment:
Suuuuuure about that `key`? I know I'm extra cautious around keys, but IMO anything that's user-driven can (and will) have duplications. You don't even need a weird use-case for it: don't forget that this is here driving the preview in studio, too. It's very likely that someone looking to add another server will start by duplicating an existing one. At that point you are in undefined behavior mode. 

I propose we err on the safe side and use indexes for most things, unless there's a native ID (I mean like a postrgres key, something that's guaranteed by our API to be unique).